components/core/src/clp_s/search/OutputHandler.hpp

```diff
@@ -187,6 +197,50 @@ class ResultsCacheOutputHandler : public OutputHandler {
             QueryResultGreaterTimestampComparator>
             m_latest_results;
 };
+
+/**
+ * Output handler that performs an aggregation operation and can send the results to a reducer
+ */
+class ReducerOutputHandler : public OutputHandler {
```

Comment:
Why do we need `ReducerOutputHandler`?

Reply:
We don't necessarily need it. I had a vague idea that I would inherit from it when adding support for timeline aggregations, but I guess it doesn't really do enough to justify its existence -- I'll remove it.